modules/channel-smooch/src/backend/client.ts

```diff
@@ -0,0 +1,128 @@
+import * as sdk from 'botpress/sdk'
+import Smooch from 'smooch-core'
+
+import { Config } from '../config'
+
+import { Clients } from './typings'
+
+const name = 'smooch.sendMessage'
```

Comment:
Name of ? When declaring a constant, please give a meaningful name, also use all caps when declared outside of the class